roles/usb_lib/files/upload2usb/upload2usb.php

```diff
@@ -29,14 +31,30 @@ function getTargetUSBDriveLocation () {
              ];
              throw new RuntimeException(json_encode($exception_data));
          } elseif ($rmv_usb_paths_count > 1) {
+             //look for second mountpoint
+             $second_mount = shell_exec('mount | grep udisks |  cut -d " " -f 3');
+             if (!empty($second_mount)) {
+                 // should test for usbbooted machines and exclude /
+                 $mnt_point = ($second_mount);
+                 shell_exec('pumount $second_mount');
+               }
+
+             $second_mount2 = shell_exec('mount | grep udisks |  cut -d " " -f 3');
```

Comment:
Just lint suggestions...

```suggestion
             $second_mount = shell_exec('mount | grep udisks | cut -d " " -f 3');
             if (!empty($second_mount)) {
                 // should test for usbbooted machines and exclude /
                 $mnt_point = ($second_mount);
                 shell_exec('pumount $second_mount');
               }

             $second_mount2 = shell_exec('mount | grep udisks | cut -d " " -f 3');
```

Reply:
I'll commit after the tests run.